ephios/core/services/notifications/backends.py

```diff
@@ -32,6 +33,8 @@ def send_all_notifications():
         for notification in Notification.objects.filter(failed=False):
             if backend.can_send(notification) and backend.user_prefers_sending(notification):
                 try:
+                    if notification.user:
+                        translation.activate(notification.user.language)
                     backend.send(notification)
```

Comment:
I thought this would be a context manager...

Reply:
the django docs don't say that this is possible, they just set the language back to the previous value if needed, but I figured that wouldn't be necessary here